tensorflow/python/keras/saving/hdf5_format.py

```diff
@@ -35,7 +35,7 @@
 from tensorflow.python.platform import tf_logging as logging
 from tensorflow.python.util import serialization
 from tensorflow.python.util.lazy_loader import LazyLoader
-import tensorflow.python.platform.gfile as gfile
+from tensorflow.python.platform.gfile import GFile, Remove, Exists
```

Comment:
Apologies. I just realized `Remove` and `Exists` are not methods of the `GFile` class. Let's revert the import change again. Apologies for not catching this before and asking you to do a change that was not needed.